verify/v1/verify.yaml

```diff
@@ -487,7 +487,7 @@ paths:
           content:
             application/json:
               schema:
-                anyOf:
+                oneOf:
                   - $ref: '#/components/schemas/VerifyResponse'
                   - $ref: '#/components/schemas/ValidNumberFormatFalseResponse'
```

Comment:
with oneOf, can we see the rest of responses in actual conversation?

Reply:
no but this should be one of...
oneof means that only one of the responses will be sent back
anyof means that multiple of the responses can be sent back

https://swagger.io/docs/specification/data-models/oneof-anyof-allof-not/

So I think oneof is the correct option